smoke_test/helpers/http_status_validation.rb

```diff
@@ -0,0 +1,16 @@
+module HttpStatusValidation
+module_function
+
+  def validate_response_status!
+    unless page.status_code == 200
+      puts <<-EOF
+Step Failed: Expected status 200, got status #{page.status_code}
+Step Failed: Request path was: #{page.current_path}
+      EOF
```

Comment:
It's not rails :(

I didn't want in add a dependency for active_support just for that.